koku/sources/kafka_listener.py

```diff
@@ -149,39 +167,20 @@ def _sources_network_info_sync(source_id, auth_header):
 
     if source_type_id == 1:
         source_type = 'OCP'
-        authentication = source_details.get('uid')
+        authentication = {'resource_name': source_details.get('uid')}
     elif source_type_id == 2:
         source_type = 'AWS'
-        authentication = sources_network.get_aws_role_arn()
+        authentication = {'resource_name': sources_network.get_aws_role_arn()}
+    elif source_type_id == 3:
```

Comment:
It might be good to move these magic ID numbers into a set of constants (or a list/dict of constants).

Reply:
Good idea.  Added some constants